app/managers.py

```diff
@@ -14,8 +16,8 @@ def create(self, first_name: str, last_name: str) -> None:
         self._connection.commit()
 
     def all(self) -> list:
-        actors = self._connection.execute(f"SELECT * FROM {self._table}")
-        result_list = [str(actor) for actor in actors]
+        actor_line = self._connection.execute(f"SELECT * FROM {self._table}")
+        result_list = [Actor(*filed) for filed in actor_line]

```

Comment:
what variable name `filed` mean? Please, use descriptive variable names (actor_data for example)